### DISCUSSION TITLE
Are you using Paketo Buildpacks?

### DISCUSSION THREAD

**Discussion by davidmirror-ops** [paketo-buildpacks/feedback#30] [Show and tell]
If you are using Paketo Buildpacks, first of all, we'd like to say:

**Thank you!**

We are always interested in finding out who is using Paketo Buildpacks, what attracted you to use it and, if you are interested, help promote your organization

Let us know by adding a comment to this Discussion including:

1. Brief description of your use case with Paketo Buildpacks 
2. Link to the website/documentation where the usage of Paketo Buildpacks is directly mentioned
 
This information will be incorporated into the [ADOPTERS.md](https://github.com/paketo-buildpacks/community/blob/main/ADOPTERS.md) file in this repo for others to see.

You can also open a pull request adding yourself to `ADOPTERS.md`

**Comment by lance** [comment]
Hi there. The Knative project uses Buildpacks for its Functions component. Since Functions were only a recent addition to Knative Core, the documentation still resides in the function repo.

1 Knative
2 https://knative.dev https://github.com/knative-sandbox/kn-plugin-func
3 Knative is global :) 
4 Reach out to us on the Knative Slack at https://knative.slack.com/archives/CUU6LEMRT
5 Paketo buildpacks serve as the default method of converting a Knative function into an OCI container for deployment. The `func` CLI uses the `pack` Go library to programatically invoke a build, potentially applying additional buildpacks when doing so.
6 I don't have an SVG, but here's a PNG https://knative.dev/docs/images/logo/rgb/knative-logo-rgb.png

**Comment by fgrehm** [comment]
Hey peeps!

At Doximity are extending paketo stacks and using ruby, go and nodejs buildpacks as the foundation of our internal platform. We don't have much shared publicly, besides these 2 blog posts from a former employee:

* https://technology.doximity.com/articles/buildpacks-vs-dockerfiles
* https://technology.doximity.com/articles/writing-a-buildpack-for-the-rails-asset-pipeline

The platform is becoming more stable now though. We're in the process of ramping up and migrating away from our previous infra so hopefully we'll have more to share about it moving forward. I was going to open a PR listing us as adopters but not sure where I'd link it to :sweat_smile: if you have any suggestions LMK

**Reply by davidmirror-ops** [reply]
Hey @fgrehm thank you for letting us know!
I think either of the blog posts are just fine for the list, we'll get you added!

Please let us know if there's anything we can do better to help your use cases, definitely would be awesome to hear about your learnings so far

**Comment by jhcp** [comment]
Hi! At Randoli we are glad to use Paketo so that our users can easily containerize their apps. Here's the info for the adopters list, if you could please add us:

Link to Paketo's mention (Randoli): https://randoli.ca/app-director
Description: App Director users can choose Paketo buildpacks to containerize their applications and then deploy them to any Kubernetes cluster straight from the UI.

Thank you for the great work on cloud native buildpacks!

**Comment by fg-j** [comment]
Welcome to the Adopters list, @jhcp and @fgrehm ! https://github.com/paketo-buildpacks/community/blob/main/ADOPTERS.md

**Comment by Adam262** [comment]
Hey there! Here's info for the adopters list. Thanks for your work!

Greenhouse uses the Paketo Python and NodeJS buildpacks to drive developer builds within our internal Platform as a Service. We used to only use buildpacks we built and maintained ourselves. Switching to Paketo has been pretty seamless and saved us a ton of work. The support is great and we're able to supplement at times with small downstream buildpacks. We intend to continue switching to upstream - first Golang and next Ruby.

**Comment by findajay** [comment]
Hello Hello, Thanks for supporting this awesome work.

At Apcoa we use Paketo build packs for our azure native app services build on Linux using .Net. Paketo has been really helpful in managing infrastructure automation while developing fast from developer point of view. We are also introducing few more services like Az functions and would like them also built with Paketo build packs.  Do Paketo already have a stack for Azure functions ?

**Reply by sophiewigmore** [reply]
Hey @findajay 👋 I'm glad to hear buildpacks have been working well for your use case! We don't currently have an Azure functions related stack in the project, but someone else in our community may have some ideas.. We're reachable for further discussion on paketobuildpacks.slack.com

**Reply by sophiewigmore** [reply]
I do recall a question from @dashaun on the subject of .NET/ Azure functions a while ago but I'm not sure what the resolution was, perhaps he has some insights?
